src/client/screens/Feed/index.js

```diff
@@ -193,6 +194,36 @@ class FeedPage extends React.Component {
     }
   }
 
+  handleErrorDelete(xhr, _id) {
+    if (xhr.status === HTTP_STATUSES.OK) {
+      this.setState({deleteKey: this.state.deleteKey.concat(_id)});
+    } else if (xhr.status === HTTP_STATUSES.UNAUTHORIZED) {
+      this.setState(
+        {errorMessage: 'Sorry, you are not authorized, please log in first!'}
+      );
+    } else {
+      this.setState(
+        {errorMessage: 'Sorry, Server Error! Please try again later!'}
+      );
+    }
+  }
+
+  deletePost(event, item) {
+    let xhr = new XMLHttpRequest();
+    let token = window.localStorage.getItem('token');
+
+    xhr.addEventListener('readystatechange', function() {
+      if (xhr.readyState === XMLHttpRequest.DONE) {
+        this.handleErrorDelete(xhr, item._id);
+      }
+    }.bind(this));
+    xhr.open('DELETE', '/api/post/' + item._id);
+    xhr.setRequestHeader('Accept', 'application/json');
+    xhr.setRequestHeader('Content-Type', 'application/json');
+    xhr.setRequestHeader('Authorization', token);
+    xhr.send(null);
```

Comment:
why do send anything in the body? if there is no body, you dont need to pass anything as parameter